apis/lambda/v1alpha1/generator-config.yaml

```
@@ -0,0 +1,9 @@
+ignore:
+  field_paths:
+    - CreateFunctionInput.VPCConfig.SecurityGroupIDs
+    - CreateFunctionInput.VPCConfig.SubnetIDs
+    - UpdateFunctionInput.VPCConfig.SecurityGroupIDs
+    - UpdateFunctionInput.VPCConfig.SubnetIDs
+  resource_names:
+    - Alias
+    - EventSourceMapping
```

```suggestion
    - EventSourceMapping
    - CodeSigningConfig
```

Seems we need `CodeSigningConfig` to be ignored as well. Note that ignore doesn't delete already generated files, so you'd need to manually remove `pkg/controller/lambda/codesigningconfig` and related files in `apis/lambda`